docs/sources/enterprise/access-control-refactor/about-fgac.md

```diff
@@ -0,0 +1,105 @@
+
+- `fixed:datasources:id:reader`: Enables the viewer to see the ID of a data source.
+- `fixed:organization:reader`: Enables the viewer to see a list of organizations.
+- `fixed:annotations:reader`: Enables the viewer to see annotations that other users have added to a dashboard.
+- `fixed:annotations.dashboard:writer`: Enables the viewer to add annotations to a dashboard.
+
+For more information about the fixed role associated with each built-in role, refer to [Built-in role definitions]({{< relref "./built-in-role-definitions.md" >}}).
+
+> **Note:** You must assign each Grafana user a built-in role.
+
+## Fixed roles
+
+Grafana Enterprise includes the ability for you to assign discrete fixed roles to users and teams, which enables you fine-grained control over user permissions. Grafana Labs manages fixed role definitions.
+
+Assign fixed roles when the built-in roles do not meet your permission requirements. For example, you might want the built-in viewer role to also edit dashboards. Or you might want the editor role to also create and manage users. To meet your needs, you can use fixed roles to extend built-in roles and provide users more granular access to create, view, and update Grafana resources.
+
+### Resources with fine-grained permissions
+
+Fine-grained access control is available for the following capabilities:
+
+- [Annotations]({{< relref "../../dashboards/annotations.md" >}})
+- [API keys]({{< relref "../../administration/api-keys/_index.md" >}})
+- [Dashboards]({{< relref "../../dashboards/_index.md" >}})
+- [Data sources]({{< relref "../../datasources/_index.md" >}})
+- [Explore]({{< relref "../../explore/_index.md" >}})
+- [Folders]({{< relref "../../dashboards/dashboard_folders.md" >}})
+- [LDAP]({{< relref "../../auth/ldap/_index.md" >}})
+- [Licenses]({{< relref "../license/_index.md" >}})
+- [Organizations]({{< relref "../../administration/manage-organizations/_index.md" >}})
+- [Provisioning]({{< relref "../../administration/provisioning/_index.md" >}})
+- [Reports]({{< relref "../reporting.md" >}})
+- [Roles]({{< relref "../../administration/manage-users-permissions/_index.md" >}})
+- [Settings]({{< relref "../settings-updates.md" >}})
+- [Service accounts]({{< relref "../../administration/service-accounts/_index.md" >}})
+- Statistics
+- [Teams]({{< relref "../../administration/manage-users-permissions/manage-teams/_index.md" >}})
+- [Users]({{< relref "../../administration/manage-users-and-permissions/manage-server-users/_index.md" >}})
+
+To learn more about the permissions associated with each fixed role, refer to [Fine-grained access control fixed role definitions]({{< relref "./rbac-fixed-role-definitions.md" >}}).
+
+To learn how to assign fixed roles to a user or team, refer to [Assign and manage fixed roles]({{< relref "./assign-manage-fixed-roles.md" >}})
```

Comment:
One quick question here--is the standard to use "to learn more about", "to learn how to" or "for more information on" - just a question for my learning, really. It's an easy sentence to standardize across docs, too.